docs/reference/sql/language/data-types.asciidoc

```diff
@@ -24,7 +24,6 @@ s|SQL precision
 | <<number, `half_float`>>                 | half_float      | FLOAT       | 3
 | <<number, `scaled_float`>>               | scaled_float    | DOUBLE      | 15
 | <<keyword, `keyword`>>                   | keyword         | VARCHAR     | 32,766
```

Comment:
Does the SQL `keyword` type now refer to _all_ types in the keyword family? If so, I would also make this change:
```suggestion
| <<keyword, keyword type family>>         | keyword         | VARCHAR     | 32,766
```

Reply:
Thanks, @jrodewig, that's a good suggestion, since yes, all types in that family will now be reported as `keyword`.